data-model/fdo-type/machine-annotation-service/0.3.0/schema/machine-annotation-service-secret.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/developer-schema/machine-annotation-service/0.3.0/machine-annotation-service-secret.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment": "Machine Annotation Service Environment Version 0.3.0",
+  "title": "MachineAnnotationServiceSecret",
+  "type": "object",
+  "description": "Remote secrets to supply to the MAS",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "The name of an environmental variable stored remotely",
+      "examples": [
+        "database.password",
+        "keycloak.secret"
+      ]
+    },
+    "secretKeyRef": {
```

Comment:
I wonder if this is necessary. I know we had it initially but then I was also wondering. Wouldn't it be possible to make this easier and have them just supply a list of key's (so the names) we then assume that the key is in the mas-secrets store and the key is something we generate for example?

Reply:
was wondering the same. i don't think it's necessary right now, but maybe if we wind up having multiple secret stores down the road it could be useful to have it like this? can change it for now.